scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -238,6 +268,7 @@ trait ClassLikeSupport:
     val name = methodKind match
       case Kind.Constructor => "this"
       case Kind.Given(_, _) => methodSymbol.name.stripPrefix("given_")
+      case Kind.Extension(_) => methodSymbol.name.stripPrefix("extension_")
```

Comment:
This should no longer be necessary, the encoding of extension methods has changed and their names AFAIU are no longer prefixed with `extension_`.

Reply:
In the version of `master` that this PR was based on, there were still `extension_` prefixes before every extension.